src/index.html

```diff
@@ -1,13 +1,47 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac-image"
+        class="card__image">
+      <div class="card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
```

Comment:
```suggestion
    <div class="card" data-qa="card">
      <img
        src="./images/imac.jpeg"
        alt="imac-image"
        class="card__image">
        
      <div class="card__description">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </div>
```
add blanks between multiline elements

Reply:
fix all such cases

Reply:
check good example in the checklist